Bottom: e71901eb2ef756ed3bd1440827c8ec4a2219914f
Top:    595d16b81a0c4e92b953604054118d2e7544a3e5
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-12 19:50:16 +0800

Refresh of autofs-5.0.5-wait-for-master-map-available-at-start.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 8a3acf7..293d8dc 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1871,7 +1871,6 @@ static int convert_log_priority(char *priority_name)
 	return -1;
 }
 
-<<<<<<< current
 static void remove_empty_args(char **argv, int *argc)
 {
 	int next_to_last = *argc - 1;
@@ -1898,7 +1897,8 @@ static void remove_empty_args(char **argv, int *argc)
 		}
 	}
 	*argc = j;
-=======
+}
+
 static int do_master_read_master(struct master *master, time_t age, unsigned int wait)
 {
 	sigset_t signalset;
@@ -1935,7 +1935,6 @@ static int do_master_read_master(struct master *master, time_t age, unsigned int
 	sigprocmask(SIG_BLOCK, &signalset, NULL);
 
 	return ret;
->>>>>>> patched
 }
 
 int main(int argc, char *argv[])
